infrastructure/helm/quantumserverless/values.yaml

```diff
@@ -73,63 +72,39 @@ manager:
 
 
 # ===================
-# Ray configs
+# Ray Cluster
 # ===================
 
-# RAY CLUSTER
-
 rayClusterEnable: true
-
 ray-cluster:
-  image:
-    repository: "qiskit/quantum-serverless-ray-node"
```

Comment:
We need base image to be ours, otherwise serverless programs will be missing dependencies  on ray nodes

Reply:
Fixed in [a8c818a](https://github.com/Qiskit-Extensions/quantum-serverless/pull/107/commits/a8c818a9a27b26f7837c69ed8628527e9b07e2d2) thanks for the catch!